test/hcal_digi_pipeline_test_config.py

```diff
@@ -0,0 +1,36 @@
+
+from LDMX.Framework import ldmxcfg
+
+# Create a process
+p = ldmxcfg.Process( 'test_hcal_digis' )
+p.libraries.append("libHcal.so")
```

Comment:
This line _shouldn't_ be needed. We want the end user to never have to look at `p.libraries`, so remove this line and make sure everything still works.